CMakeLists.txt

```diff
@@ -123,10 +123,10 @@ message (STATUS "Arch: ${arch}")
 
 if (${arch} MATCHES "x86_64")
 	#set compiler flags here for x86_64
-	set (cc_flags "")
+        set (cc_flags "-ansi")
```

Comment:
Sorry about complaining over that flag, should have tried compiling with C++11 myself before.
So here's what's happening: GCC 6 changed the C++ default from gnu++98 to gnu++14. The -ansi flag reverts this change and explicitly states the intended C++ version.
I would prefer that the flags are set using the following code snippet just after the if containing the set cc_flags commands.
<pre>
# make sure to use C++98
SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ansi")
</pre>

Another option would be to use the commits from my pull request #53 which adds support for both C++ versions.